dash/development/dcc_init.template

```diff
@@ -0,0 +1,73 @@
+"""This package provides the core React component suite for Dash."""
```

Comment:
can't this live in the dcc repo?

Reply:
this was added by accident, out of the scope of this PR.

but if we need a similar process for DCC, the tools should live here like the component generator and that template fills the `init` as well.